wnfs/src/private/directory.rs

```diff
@@ -1002,6 +1002,10 @@ impl PrivateDirectory {
         }
     }
 
+    pub fn entries<'a>(self: &'a Rc<Self>) -> impl Iterator<Item = &'a String> {
+        self.content.entries.iter().map(|x| x.0)
+    }
```

Comment:
Can you add a tiny doc to the function? 
Nit: Can you also rename the function `get_entries` just to be consistent with the rest of the code?

Reply:
:+1: